k8s/config/stage.sh

```diff
@@ -20,6 +20,7 @@ export APP_MEMORY_LIMIT=4Gi
 export APP_MEMORY_REQUEST=2Gi
 export APP_GUNICORN_WORKERS=2
 export APP_HOST=developer-portal-dev.mdn.mozit.cloud
+export APP_EXPORTED_SITE_HOST=developer-portal-published.dev.mdn.mozit.cloud
```

Comment:
Note: This URL doesn't exist, but it makes sense that it's what we'd use if/when the EKS stage (and a relevant, separate S3 bucket) is put back into play.